### DISCUSSION TITLE
Factory sound bank for hacktribe?

### DISCUSSION THREAD

**Discussion by BKLronin** [bangcorrupt/hacktribe#23] [Ideas]
Thought about making a small factory sound set with analog synths and some real instruments I have. Maybe as a starting point.

**Comment by bangcorrupt** [comment]
I have some public domain drum samples I can add to this.  A set of patterns demonstrating new feature could be a good idea too.

**Reply by BKLronin** [reply]
Ok cool, I will collect some. Got some drums made with bitwig grid recently. Still refining but if done they are kinda unique and bass guitar etc is always useful.

**Comment by BKLronin** [comment]
I added my samples in a fork. Including a demo song. Its CC0 licensed. @bangcorrupt

**Reply by bangcorrupt** [reply]
Thank you for making samples.  I created a [hacktribe-content](https://github.com/bangcorrupt/hacktribe-content) repo to keep things organised.  If you add samples to that and submit a PR I can merge.

**Reply by BKLronin** [reply]
Copy, will do.

Glad I can help somehow.

**Comment by Forumlivii** [comment]
> Ho pensato di creare un piccolo suono di fabbrica con sintetizzatori analogici e alcuni strumenti reali che ho. Forse come punto di partenza.

hello, as synths lack the various Roland: the juno

**Comment by Forumlivii** [comment]
Volendo molte cose si possono prendere dalla libreria Sample From Mars
